content/actions/learn-github-actions/environment-variables.md

```diff
@@ -124,6 +124,8 @@ Any new environment variables you set that point to a location on the filesystem
 
 The default environment variables that {% data variables.product.prodname_dotcom %} sets are available to every step in a workflow. 
 
+{% data reusables.actions.environment-variables-are-fixed %} For more information about setting environment variables, see "[AUTOTITLE](/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow)" and "[AUTOTITLE](/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable)."
```

Comment:
The `environment-variables.md` file is now called `variables.md`. After merging main from upstream into your issue-19626 branch you can make the above insertion into the same place in the `variables.md` file and delete `environment-variables.md`.